template.rb

```diff
@@ -104,7 +104,19 @@ def add_template_repository_to_source_path
 
 say "=> Copying git configuration"
 directory "lib/install/.github", ".github"
-copy_file "lib/install/.gitattributes", ".gitattributes"
+gitattributes = <<-CODE
+  *.gemspec diff=ruby
+  *.rake    diff=ruby
+  *.rb      diff=ruby
+  *.js      diff=javascript
+
+  db/schema.rb merge=ours diff=ruby
+  yarn.lock merge=ours
+  Gemfile.lock merge=ours linguist-generated
+
+  vendor/* linguist-vendored
```

Comment:
You can remove this, it’s already in the target file

Reply:
Done